.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "modules/llama_cpp_plugin/third_party/llama.cpp"]
+	path = modules/llama_cpp_plugin/third_party/llama.cpp
+	url = https://github.com/vshampor/llama.cpp
```

Comment:
I used to need the fork because llama.cpp does not have separate CMake targets for various backend builds; every build type has the same `libllama.so` output, and the name doesn't seem to be configurable via CMake options. At the same time I needed that to have both `LLAMA_CPP` and `LLAMA_CPP_CUDA` plugins as separate entities, so I needed ideally differently named `libllama.so` binaries for CPU and CUDA backends, which I made configurable in the fork. We are currently limiting ourselves to CPU backend out of the box, I think, so I don't need the fork really. Maybe a `patchelf` or specific CMake idiom will help instead. Will try with FetchContent.

Reply:
Done